File: final/final7.py
# Collection
MongoCollection albums = database.getCollection("albums");
# Images
MongoCollection imgColl = database.getCollection("images");
FindIterable images = imgColl.find();
for (Document image : images) {
	Document album = albums.find(new Document("images", image.get("_id"))).first();
	if (album == null) {
		print image;
	}
}

File: final/final7.js
var cur = db.images.find();
 
var j = 0;
while(cur.hasNext()){
	doc = cur.next();
	image_id = doc._id
 
	b = db.albums.find({images : image_id}).count()
	if(b == 0){
		db.images.remove({_id:image_id})
                print(image_id)
		j++;
	}
}
print(j)
